.php_cs.dist

```diff
@@ -7,11 +7,18 @@ $finder = (new PhpCsFixer\Finder())
 ;
 
 return (new PhpCsFixer\Config())
+    ->setRiskyAllowed(true)
     ->setRules([
         '@Symfony' => true,
-        'array_syntax' => ['syntax' => 'short'],
+        '@Symfony:risky' => true,
+        '@PHP80Migration' => true,
+        '@PHP80Migration:risky' => true,
+        '@PHPUnit84Migration:risky' => true,
         'ordered_imports' => true,
-        'declare_strict_types' => false
+        'declare_strict_types' => false,
```

Comment:
Is too much for this PR to add `declare_strict_types`?

Reply:
Maybe we can keep it for the next time?